microservice-transaction-sample/rpc/build.gradle

```diff
@@ -20,16 +20,40 @@ protobuf {
     generateProtoTasks {
         all()*.plugins { grpc {} }
     }
-    generatedFilesBaseDir = "$projectDir/src"
```

Comment:
`generatedFilesBaseDir` is deprecated in `com.google.protobuf` v0.9.2- (ref. https://github.com/google/protobuf-gradle-plugin/pull/636).